src/test/resources/pages/studentFeedbackResultsPageNUMSCALE.html

```diff
@@ -1234,7 +1234,7 @@ <h4>
           <b>
             To:
           </b>
-          Anonymous team ${participant.hash}
+          Anonymous student
```

Comment:
Is this an intended effect?

Reply:
Oh no that is a mistake. In ‘StudentFeedbackResultsPageData:182’ it should have been ‘question.giverType’ instead of ‘PARTICIPANT_TYPE_STUDENT’. There may be more test files affected by this. I will submit a fix tomorrow.